REVERT:
From 1cd557ce6873b08e66e0434a5ff8b2a2faa97f7b Mon Sep 17 00:00:00 2001
From: Lukas Oberhuber <lukaso@gmail.com>
Date: Tue, 5 Jul 2022 15:07:34 +0000
Subject: [PATCH] meson: fix export_packages for generate_gir

This commit broke graphics/libmypaint Gir generation with:
Package babl-0.1 was not found in the pkg-config search path
(there is no lib/pkgconfig/babl-0.1.pc, only babl.pc)

https://gitlab.gnome.org/GNOME/babl/-/issues/80

Index: babl/meson.build
--- babl/meson.build.orig
+++ babl/meson.build
@@ -156,7 +156,6 @@ if build_gir
     namespace: 'Babl',
     nsversion: api_version,
     header: 'babl.h',
-    export_packages: 'babl-0.1',
     install: true,
   )
 
